forest/poplar/legacy/common_funcs.py

```diff
@@ -163,61 +208,74 @@ def read_data(beiwe_id: str, study_folder: str, datastream: str, tz_str: str,
+            - {"survey_answers", "survey_timings", "audio_recordings"}
+        )
         all_timestamps: list = []
 
         for i in directories:
-            directory_filenames, directory_filestamps = get_files_timestamps(
+            _, directory_filestamps = get_files_timestamps(
                 os.path.join(study_folder, beiwe_id, i)
             )
             all_timestamps += directory_filestamps.tolist()
+
         ordered_timestamps = sorted(all_timestamps)
         stamp_end1 = ordered_timestamps[-1]
+
         if time_end is None:
             stamp_end = stamp_end1
         else:
             stamp_end2 = datetime2stamp(time_end, tz_str)
             stamp_end = min(stamp_end1, stamp_end2)
 
         # extract the filenames in range
-        files_in_range = list(filenames[
-            (filestamps >= stamp_start) * (filestamps < stamp_end)
-        ])
+        files_in_range = list(
+            filenames[(filestamps >= stamp_start) * (filestamps < stamp_end)]
+        )
         if len(files_in_range) == 0:
-            logger.warning('User %s: There are no %s data in range.',
-                           beiwe_id, datastream)
+            logger.warning(
+                "User %s: There are no %s data in range.", beiwe_id, datastream
+            )
         else:
-            if datastream != 'accelerometer':
+            if datastream != "accelerometer":
                 # read in the data one by one file and stack them
                 for data_file in files_in_range:
                     dest_path = os.path.join(folder_path, data_file)
                     hour_data = pd.read_csv(dest_path)
-                    if df.shape[0] == 0:
-                        df = hour_data
+                    if res.shape[0] == 0:
+                        res = hour_data
                     else:
-                        df = pd.concat([df, hour_data], ignore_index=True)
+                        res = pd.concat([res, hour_data], ignore_index=True)
 
     if datastream == "accelerometer":
         return files_in_range, stamp_start, stamp_end
-    else:
-        return df, stamp_start, stamp_end
 
+    return res, stamp_start, stamp_end
 
-def write_all_summaries(beiwe_id: str, stats_pdframe: pd.DataFrame,
-                        output_folder: str):
-    """
-    Docstring
-    Args: beiwe_id: str, stats_pdframe is pd dataframe (summary stats)
-          output_path should be the folder path where you want to save the
-          output
-    Return: write out as csv files named by user ID
+
+def write_all_summaries(
+    beiwe_id: str, stats_pdframe: pd.DataFrame, output_path: str
+):
+    """Write out all the summary stats for a user
+
+    Args:
+        beiwe_id: str,
+            beiwe ID
+        stats_pdframe: pd.DataFrame,
+            the summary stats for a user
+        output_path: str,
+            the path to write out the summary stats
+
+    Returns:
+        write out as csv files named by user ID
```

Comment:
It actually returns `None`. CSV files are a side effect and should go into the function description.